e2e-tests/ukef/cypress/e2e/journeys/portal/MIA-underwriter-managers-approved-decision-updates-portal-deal-and-submission-type-to-MIN/MIA-underwriter-managers-approved-decision-updates-portal-deal-and-submission-type-to-MIN.spec.js

```diff
@@ -96,9 +96,8 @@ context('Portal to TFM deal submission', () => {
     cy.clearCookie('dtfs-session');
     cy.clearCookie('_csrf');
     cy.getCookies().should('be.empty');
-    cy.forceVisit(TFM_URL);
 
-    cy.tfmLogin(UNDERWRITER_MANAGER_1);
```

Comment:
e2e-tests/ukef  has 2 cypress login commands: `portal login` and `tfm login`. Recent commit assigned same name `login` to both.